### PR TITLE
Forlorn Hope mercs, new PR, new sprites [READY]

### PR DESCRIPTION
**Pull request by Dedwolf**
## About The Pull Request

See #1069 for context, this PR just fixes the issues pointed out with Seelies and some icons. 

The new sprites are in! See below for the updated sprites, courtesy of the fantastic Cre!

t3j68GYbIxKA2yjUBwL4f3t79jdXSlhnAAAAAElFTkSuQmCC

## Why It's Good For The Game

EDITS: Changed the gambesson out for a tunic instead.
Merc slots have been increased so that every merc class is available whenever you select the mercenary job, no more clicking merc just to miss out on your preferred role.

See #1069

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/mercenaries/seelie.dm [hunk 1]
```diff
@@ -8,13 +8,13 @@
 	allowed_races = list(/datum/species/seelie,)
 	allowed_ages = ALL_AGES_LIST
 	tutorial = "You are a fae sprite that came into existance around this town, either recently or long ago. Perhaps you support people in their trade and craft, or you may be a disruptive nuisance that plagues the town. Either way, your miriad of mysterious spells and abilites will support you in your endeavours."
-	outfit = /datum/outfit/job/roguetown/mercenary/seelie
+	outfit = /datum/outfit/job/roguetown/seelie
 	display_order = JDO_SEELIE
-	min_pq = 30
+	min_pq = 0
```
Comment (TheFenix323) [inline code/modules/jobs/job_types/roguetown/mercenaries/seelie.dm:13]: Why was the PQ requirement for Town Seelie changed?

**Comment by Hmeister-real** [comment]
i really really fucking despise this. 

No offense to the creator, it is well done and looks cool but it is purely just powergamey and is almost undeniably the best possible pick stat and equipment wise. We do NOT need more fucking mercs, especially not overly powerul ones like this. Please, please don't let this go to waste, throw it onto Hearthstone or something, but this has no place being in Ratwood IMO

**Comment by Hmeister-real** [comment]
> Let me know in what ways they're OP and powergamey and i'll rectify it.

They start with overly better armor than any other mercs, it is almost fully body and that is extremely stupid, alongside having skills in the most meta weapon and having a new essentially meta weapon on spawn.

> Nothing wrong with having more mercs.

We already have 3 entire other mercs, we do NOT need any more. It is just bloat. we need more insentives to be playing key roles in the Keep, Garrison and Church, we do NOT need extremely powerful external roles to further incentivise not going as key roles...

**Comment by Hmeister-real** [comment]
after trying it, get rid of the Gorget and the warhammer, those are absurdly powergamey. I would say to remove the mask too, but i guess thats their whole style, but it is stupidly strong covering the ENTIRE head and also being steel. I just really don't like these mercs, they are stupid

**Comment by HissyNays** [comment]
Not sure if already possible. But if not, a humble request, please allow drakians and sisseans to be forlorn mercenary.

I was genuinely hyped for this because I wanted a merc role thats not a shady desert rider for my Noc worshipping lizards.
So I see a Noc themed mercenary added and find that my most favorite races aren't in allowed list. 

My disapointment was immesureable and my day was ruined.

**Comment by HissyNays** [comment]
> > > Not sure if already possible. But if not, a humble request, please allow drakians and sisseans to be forlorn mercenary.
> > > I was genuinely hyped for this because I wanted a merc role thats not a shady desert rider for my Noc worshipping lizards. So I see a Noc themed mercenary added and find that my most favorite races aren't in allowed list.
> > > My disapointment was immesureable and my day was ruined.
> > 
> > 
> > maybe, had a fair few complaints about other races aside from lupian getting access to it, like tabaxi, not sure if I should restrict it or open it up more.
> 
> Keeping it restricted to certain species emphasizes the company's own identity, which is good.
> 
> I think it's fine where it stands. Lupians and Vulpkanin being the primary intended demographic, with wild-kin available for those who play other sorts of creatures. Not sure what else would fit, given they're canine-themed with their helmet and all.
> 
> Sisseans would look out of place for sure imo, not so certain about Tabaxi though.

Canine helmet being the more fitting to lizards, because there's no dragon helmet, there'e only flat face human helmets as alternative.

**Comment by Hmeister-real** [comment]
What is the reasoning for this change?

image

**Comment by HissyNays** [comment]
> > > Not sure if already possible. But if not, a humble request, please allow drakians and sisseans to be forlorn mercenary.
> > > I was genuinely hyped for this because I wanted a merc role thats not a shady desert rider for my Noc worshipping lizards. So I see a Noc themed mercenary added and find that my most favorite races aren't in allowed list.
> > > My disapointment was immesureable and my day was ruined.
> > 
> > 
> > maybe, had a fair few complaints about other races aside from lupian getting access to it, like tabaxi, not sure if I should restrict it or open it up more.
> 
> Keeping it restricted to certain species emphasizes the company's own identity, which is good.
> 
> I think it's fine where it stands. Lupians and Vulpkanin being the primary intended demographic, with wild-kin available for those who play other sorts of creatures. Not sure what else would fit, given they're canine-themed with their helmet and all.
> 
> Sisseans would look out of place for sure imo, not so certain about Tabaxi though.

Wild-kin though... is extremely out of place.

**Pull request merged**